### PR TITLE
Removing all helm commands from the build process.

### PR DESCRIPTION
**Pull request by ybettan**
Those helm commands are needed to manipulate helm charts used by the
`file:///...` url, which was agreed by partners to be removed.

The build process of OCP (including SRO in it) is done in a disconnected
env, therefore, we cannot install the `helm` package during the build
process .

Also removing partners charts.

Signed-off-by: Yoni Bettan <yonibettan@gmail.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -14,12 +14,6 @@ RUN ["go",  "mod", "download"]
 COPY Makefile* ./
 RUN ["make", "controller-gen"]
 
-COPY cmd/ cmd/
-RUN ["make", "helm-plugins/cm-getter/cm-getter"]
```
Comment (qbarrand) [inline Dockerfile:18]: Don't we need this?

Comment (ybettan) [reply Dockerfile:18]: We DO, deleted it my mistake.
Thanks, it is now fixed.

**Pull request merged**